src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -1,6 +1,20 @@
 // global parameter to remember settings for custom access level
 
-var INSTRUCTOR_COURSE_EDIT_INSTRUCTOR_ACCESS_LEVEL_WHEN_LOADING_PAGE = [];
+var instructorCourseEditInstructorAccessLevelWhenLoadingPage = [];
+var instructorCourseEditDefaultPrivilegeValuesForCustomRole = [];
```

Comment:
I think it will be good if you added more explanations as comments on what these 2 variables are. In particular, it may not be clear as to why an array is needed for <code>instructorCourseEditDefaultPrivilegeValuesForCustomRole</code>